src/data.rs

```diff
@@ -60,6 +67,24 @@ impl Data {
             Data::PrivAppendable(ref data) => data.identifier(),
         }
     }
+
+    /// Validate data size.
+    pub fn validate_size(&self) -> bool {
+        let serialised_data = match serialisation::serialise(&self) {
```

Comment:
That shouldn't be as much of a problem once we have the new version of utils published and can use [`serialised_size()`](https://github.com/Fraser999/maidsafe_utilities/blob/master/src/serialisation.rs#L109-L112), right?